sormas-e2e-tests/src/main/java/org/sormas/e2etests/services/api/EventApiService.java

```diff
@@ -43,9 +51,25 @@ public Event buildGeneratedEvent() {
         .eventTitle(String.valueOf(System.currentTimeMillis()))
         .startDate(new Date())
         .reportDateTime(new Date())
-        .eventLocation(EventLocation.builder().uuid(UUID.randomUUID().toString()).build())
         .riskLevel(RiskLevelValues.getRandomRiskLevelName())
         .typeOfPlace(TypeOfPlace.getRandomTypeOfPlace())
+        .eventManagementStatus("ONGOING")
```

Comment:
get data from Enums, and also, if possible, vary it instead of using only ONGOING

Reply:
added dedicted Enum. In terms of randomizing this field, from last experience with filling entities fields with random enum values I don't feel like it is a good approach, especially for testing filter purposes.